contracts/v2/PolygonRollupManager.sol

```diff
@@ -237,7 +237,7 @@ contract PolygonRollupManager is
         keccak256("EMERGENCY_COUNCIL_ADMIN");
 
     // Current rollup manager version
-    string public constant ROLLUP_MANAGER_VERSION = "al-v0.3.1";
+    string public constant ROLLUP_MANAGER_VERSION = "al-v0.3.2";
```

Comment:
@laisolizq to propose new versioning on the contract.
Contract is already on production, so `v1` is mandatory.
If we consider previous version as `v1.0.1` (the one with `initMigration()`)....this one should be `v1.1.0` since it includes a breaking change for `VerifierType:Pessimistic`

Reply:
My proposal is that the current version should be `v2.1.0`, so with this change it would be `v2.1.1`.
You can see my proposal here: https://github.com/agglayer/agglayer-contracts/pull/502

Reply:
This PR will be merged before the versioning restrictions so they will be applied on next updates